gsw/geostrophy.py

```diff
@@ -295,7 +295,7 @@ def geostrophic_velocity(geo_strf, lon, lat, p=0, axis=0):
 
     if lon.shape != lat.shape or lon.ndim != 1:
         raise ValueError('lon, lat must be 1-D and matching; found shapes'
-                         ' %s and %s' % (lon.shape, lat.shape))
+                         ' {} and {}'.format(lon.shape, lat.shape))
```

Comment:
Might as well go all the way to an f-string.

Reply:
Will do. I'm not sure why pyupgrade, even though it is set to py38, stopped at `.format`.